tasks/93_make-a-person/make-a-person.js

```diff
@@ -0,0 +1,32 @@
+export class Person {
+  constructor(fullName) {
+    this.fullName = fullName;
+    const dividedFullName = fullName.split(' ');
+    this._firstName = dividedFullName[0];
+    this._lastName = dividedFullName[1];
+  }
+
+  getFirstName() {
+    return this._firstName;
+  }
+
+  getLastName() {
+    return this._lastName;
+  }
+
+  getFullName() {
+    return this._firstName + ' ' + this._lastName;
+  }
+
+  setFirstName(first) {
+    this._firstName = first;
+  }
+
+  setLastName(last) {
+    this._lastName = last;
+  }
+
+  setFullName(fullName) {
+    this.fullName = fullName;
+  }
+}
```

Comment:
You have dependencies: 

1. first name depends on full name and vice versa. If you set first name, you should update fullName as well and vice versa
2. last name depends on full name and vice versa. If you set last name, you should update fullName as well and vice versa

unfortunately, tests are not covered this case. But it's real production case. It is too hard to keep up to date both source of truth, so please try to **have only one source of truth**. For example, only first + last name in this class

**steps to reproduce**: set  another full name, get first name.
**expected result**: first name should be updated
The same for second name.

Reply:
And partial test does not work
![image](https://user-images.githubusercontent.com/11594423/57847106-f1348080-77d5-11e9-985d-0862771a7c18.png)